job/pytorchtrainingjob/ddp/torch_mnist_trainingjob.py

```diff
@@ -124,6 +124,7 @@ def test(val=False, epoch=None):
     rank = dist.get_rank()
     world_size = dist.get_world_size()
 
+    # rank 0 download datasets in advance
```

Comment:
1. downloads
2. this comment needs to be moved to right before line 130?

Reply:
modifed and moved